user/pages/02.development/03.creating-a-skill/docs.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Development Setup
+taxonomy:
+    category:
+        - docs
+---
+
+# Setting up the folders
+
+Once you've come up with your idea for a skill, make a new folder in `mycroft-core/mycroft/skills` with your desired skill name.
+

```

Comment:
Please, mention the folder's name must be **lower-case** and differ from the most famous python modules like `time` to avoid conflicts.